### PR TITLE
feature: add gasPrice option or ethersOptions argument

### PR DESCRIPTION
**Pull request by jtakalai**
Fixes one part of #255

### CODE REVIEW

**Comment by jtakalai** [comment]
Tests look ok  ;D

File: src/dataunion/DataUnion.ts [hunk 1]
```diff
@@ -506,13 +507,13 @@ export class DataUnion {
     /**
      * Admin: set admin fee (between 0.0 and 1.0) for the data union
      */
-    async setAdminFee(newFeeFraction: number) {
+    async setAdminFee(newFeeFraction: number, ethersOptions = {}) {
```
Comment (teogeb) [inline src/dataunion/DataUnion.ts:510]: Define the parameter type for `ethersOptions` (here + `transportMessage()` + `transportSignaturesForMessage()`)

If `ethers.js` doesn't provide the interface, maybe we need define it ourself:
```
export interface EthersOptions {
    gasPrice?: BigNumber|string
    gasLimit?: BigNumber|string
    value?: BigNumber|string
    nonce?: number
}
```

File: README.md [hunk 2]
```diff
@@ -376,9 +376,9 @@ These DataUnion-specific options can be given to `new StreamrClient` options:
 | Name                              | Returns             | Description                                                    |
 | :-------------------------------- | :------------------ | :------------------------------------------------------------- |
 | createSecret(\[name])             | string              | Create a secret for a Data Union                               |
-| setAdminFee(newFeeFraction)       | Transaction receipt | `newFeeFraction` is a `Number` between 0.0 and 1.0 (inclusive) |
 | addMembers(memberAddressList)     | Transaction receipt | Add members                                                    |
 | removeMembers(memberAddressList)  | Transaction receipt | Remove members from Data Union                                 |
+| setAdminFee(newFeeFraction[, ethersOptions])                                                      | Transaction receipt     | `newFeeFraction` is a `Number` between 0.0 and 1.0 (inclusive) |
```
Comment (teogeb) [inline README.md:381]: Update the new parameter signature also for `transportMessage` (line 416)

File: src/dataunion/DataUnion.ts (hunk 1 above)
Comment (timoxley) [reply src/dataunion/DataUnion.ts:510]: agreed

**Review by timoxley — APPROVED** [review]
lgtm

File: README.md (hunk 2 above)
Comment (jtakalai) [reply README.md:381]: not sure what this means since it already has the link to ethers docs where you can see those available ethers-options

**Comment by jtakalai** [comment]
dataunion tests pass, I'll call it a success.

**Pull request merged**